packages/cloud_firestore/android/src/main/java/io/flutter/plugins/firebase/cloudfirestore/CloudFirestorePlugin.java

```diff
@@ -560,6 +561,7 @@ public void onFailure(@NonNull Exception e) {
   private static final byte DATE_TIME = (byte) 128;
   private static final byte GEO_POINT = (byte) 129;
   private static final byte DOCUMENT_REFERENCE = (byte) 130;
+  private static final byte BLOB = (byte) 131;
```

Comment:
This constant is not used, since your code below reduces the `Blob` case to the `byte[]` case.